### PR TITLE
Updated patch for Drupal 7.41 version

### PR DESCRIPTION
**Pull request by naveenkampelli**

### CODE REVIEW

**Comment by darthsteven** [comment]
@naveenkampelli Hi, so the patch looks okay in terms of the changes, but it doesn't cleanly apply to Drupal 7.41 in my testing.

I think that maybe there's an extra line break in one of the hunks. I'll add a comment where.

Does this apply cleanly for you? It's odd that diff or git diff would produce a patch that didn't apply, how did you create the patch?

File: drupal-7.41-states-toooooo-slow.patch [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+         .closest('.form-item, .form-submit, .form-wrapper').toggleClass('form-disabled', e.value)
+@@ -492,6 +492,7 @@ $(document).bind('state:disabled', function(e) {
+ 
+ $(document).bind('state:required', function(e) {
+   if (e.trigger) {
++    e.target = e.states_target || e.target; 
+     if (e.value) {
+       var $label = $(e.target).closest('.form-item, .form-wrapper').find('label');
+       // Avoids duplicate required markers on initialization.
+
```
Comment (darthsteven) [inline drupal-7.41-states-toooooo-slow.patch]: My version of git apply and patch don't like this extra line, because the hunk header isn't valid.

**Comment by naveenkampelli** [comment]
Hi @darthsteven , we are create the patch by using git diff and it's successfully getting updated for drupal 7.41 core.

**Comment by darthsteven** [comment]
@naveenkampelli Just done a little testing with my colleagues, and from what we can see this applies on linux, but not on mac. Maybe because of the different implementation of the patch and git apply commands on those platforms.

But, on a visual inspection of the patch, the line I mentioned really shouldn't be there. There should be 3 lines of context around a change, and including the blank line, there are 4 lines. If this was an additional line of context, then I'd expect to to contain the actual line from the file being patched, which isn't blank.

Can you just remove the extra blank line from your patch please, and then this is okay to be merged in.

**Comment by naveenkampelli** [comment]
Hi @darthsteven , removed the  extra line from our patch.Please look into it .

**Comment by darthsteven** [comment]
@naveenkampelli Thanks! Looks great.

**Comment by naveenkampelli** [comment]
@darthsteven  Thanks

**Pull request merged**